.travis.yml

```diff
@@ -1,55 +1,44 @@
 sudo: required
 dist: trusty
+cache: bundler
 
 addons:
   apt:
     sources:
-      - chef-current-trusty
+      - chef-stable-trusty
     packages:
       - chefdk
 
-# Don't `bundle install`
-install: echo "skip bundle install"
-
 branches:
   only:
     - master
 
 services: docker
 
 env:
-  matrix:
-  - INSTANCE=default-centos-6
-  - INSTANCE=default-centos-7
-  - INSTANCE=default-debian-7
-  - INSTANCE=default-debian-8
-  - INSTANCE=default-fedora-latest
-  - INSTANCE=default-ubuntu-1404
-  - INSTANCE=default-ubuntu-1604
-  - INSTANCE=mms-agent-centos-6
-  - INSTANCE=mms-agent-centos-7
-  - INSTANCE=mms-agent-ubuntu-1404
-  - INSTANCE=mms-agent-ubuntu-1604
+- INSTANCE=add-to-list-centos-7
```

Comment:
does this actually work?

Reply:
And this commit it why you shouldn't work late at night 😂